packages/examples/packages/jsx/jest.config.js

```diff
@@ -0,0 +1,36 @@
+const deepmerge = require('deepmerge');
+
+const baseConfig = require('../../../../jest.config.base');
+
+module.exports = deepmerge(baseConfig, {
+  preset: '@metamask/snaps-jest',
+
+  // Since `@metamask/snaps-jest` runs in the browser, we can't collect
```

Comment:
Is this still accurate?

Reply:
It's accurate for the wrong reason. We still can't collect coverage information since the Snap runs in a separate thread.